### PR TITLE
feat: add angular 15 template (#116)

### PR DESCRIPTION
**Pull request by dalenguyen**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/analogjs/analog/blob/main/CONTRIBUTING.md#-commit-message-guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Which package are you modifying?

- [ ] vite-angular-plugin
- [ ] astro-angular
- [x] create-analog
- [ ] router

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

- Ability to generate Angular 15 template

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/create-analog/template-angular-v15/src/environments/environment.prod.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const environment = {
```
Comment (brandonroberts) [inline packages/create-analog/template-angular-v15/src/environments/environment.prod.ts:1]: Delete this file

File: packages/create-analog/template-angular-v15/src/environments/environment.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+// This file can be replaced during build by using the `fileReplacements` array.
```
Comment (brandonroberts) [inline packages/create-analog/template-angular-v15/src/environments/environment.ts:1]: Delete this file

File: packages/create-analog/template-angular-v15/src/main.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import './polyfills';
+import { enableProdMode } from '@angular/core';
+import { bootstrapApplication } from '@angular/platform-browser';
+import { provideFileRouter } from '@analogjs/router';
+
+import { AppComponent } from './app/app.component';
+
+if (import.meta.env.PROD) {
+  enableProdMode();
+}
```
Comment (brandonroberts) [inline packages/create-analog/template-angular-v15/src/main.ts:10]: Delete these lines and imports as its no longer needed

```suggestion

```

File: packages/create-analog/index.js [hunk 4]
```diff
@@ -24,6 +24,11 @@ const APPS = [
         display: 'TypeScript',
         color: yellow,
       },
+      {
```
Comment (brandonroberts) [inline packages/create-analog/index.js:27]: Move this above `v14`

**Review by brandonroberts — CHANGES_REQUESTED** [review]

File: packages/create-analog/template-angular-v15/src/main.ts (hunk 3 above)
Comment (dalenguyen) [reply packages/create-analog/template-angular-v15/src/main.ts:10]: Nice catch :D

**Review by brandonroberts — APPROVED** [review]
🎉

**Comment by brandonroberts** [comment]
Thanks! @allcontributors add @dalenguyen for code

**Comment by brandonroberts** [comment]
Thanks @teobmg yes you can open a separate PR for that

**Pull request merged**